packages/liquid-html-parser/src/stage-1-cst.spec.ts

```diff
@@ -1316,6 +1316,79 @@ describe('Unit: Stage 1 (CST)', () => {
           expectPath(cst, '0.children.1.type').to.equal('LiquidDocExampleNode');
           expectPath(cst, '0.children.1.exampleContent.value').to.equal('second example\n');
         });
+
+        it('should parse @description node', () => {
+          const testStr = `{% doc %} @description {%- enddoc %}`;
+          cst = toCST(testStr);
+          expectPath(cst, '0.type').to.equal('LiquidRawTag');
+          expectPath(cst, '0.name').to.equal('doc');
+          expectPath(cst, '0.children.0.type').to.equal('LiquidDocDescriptionNode');
+          expectPath(cst, '0.children.0.content.value').to.equal('');
+        });
+
+        it('should parse a basic description tag', () => {
+          const testStr = `{% doc -%} @description {%- enddoc %}`;
+          cst = toCST(testStr);
+          expectPath(cst, '0.type').to.equal('LiquidRawTag');
+          expectPath(cst, '0.name').to.equal('doc');
+          expectPath(cst, '0.children.0.type').to.equal('LiquidDocDescriptionNode');
+          expectPath(cst, '0.children.0.content.value').to.equal('');
+        });
+
+        it('should parse @description node', () => {
+          const testStr = `{% doc %}
+          @description This is a description
+          @description This is a second description
+          {% enddoc %}`;
+          cst = toCST(testStr);
+          expectPath(cst, '0.children.0.type').to.equal('LiquidDocDescriptionNode');
+          expectPath(cst, '0.children.0.content.value').to.equal('This is a description\n');
```

Comment:
It's expected that this `\n` is included for the moment.

This is tracked [here](https://github.com/Shopify/developer-tools-team/issues/545?issue=Shopify%7Cdeveloper-tools-team%7C552)